### DISCUSSION TITLE
Omega angle. Ellipse fitting

### DISCUSSION THREAD

**Discussion by jjepastor** [mtex-toolbox/mtex#1583] [Ask Anything] [answered]
Hello Ralf and Rudiger!

I have a question about the units system  used for the "omega" angle on the function "FIT ELLIPSE". Is it given on radians or degrees?

Best regards

Pastor Juan José

**Comment by ralfHielscher** [comment, marked as answer]
It is given in radians

Ralf.

**Reply by jjepastor** [reply]
Thanks!!!
